src/Tempest/Database/QueryStatements/IndexStatement.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Database\QueryStatements;
+
+use Tempest\Database\DatabaseDialect;
+use Tempest\Database\QueryStatement;
+
+final readonly class IndexStatement implements QueryStatement
```

Comment:
I'm unsure about simple statements like this to have a separate class

Reply:
I think it's ok in this case